pipelines/pepatac.py

```diff
@@ -748,7 +763,7 @@ def main():
                 unmap_fq1, unmap_fq2 = _align_with_bt2(
                     args, tools, args.paired_end, False,
                     unmap_fq1, unmap_fq2, reference,
-                    assembly_bt2=_get_bowtie2_index(res.genomes, reference),
+                    assembly_bt2=_get_bowtie2_index(res.rgc, reference),
```

Comment:
these should be using the `res` variable instead of re-deriving the index

Reply:
these calls appear to be for prealignment assemblies, not the main `args.genome_assembly` from which the relevant bowtie value on the `res` object is derived.